.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,21 @@
+name: Devcontainer
+
+# When a pull request is opened that changes the Devcontainer configuration,
+# ensure that the container continues to build properly.
+on:
+  pull_request:
+    paths:
+      - .devcontainer/**
+    paths-ignore:
+      # The devcontainer.json isn't actually tested at the moment.
+      - .devcontainer/devcontainer.json
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: docker build .devcontainer
```

Comment:
I added a CI workflow that tests changes to the Dockerfile. It's only run on changes to `.devcontainer/`.

Reply:
it should probably also run on changes to the root README (since it contains the canonical MSRV)

Reply:
the MSRV check runs in the ci workflow, so it won't be possible for these to drift

Reply:
but isn't it possible to bump the MSRV and forget to touch the `devcontainer` path?

Reply:
If you update the MSRV in README.md, the `check-msrv` job in `ci.yml` will fail. To get that to pass, you'll have to update `.devcontainer/Dockerfile` (which will trigger this build).

Reply:
Ahh, sorry. It looked like one check was part of another job. All good then!